hopo/ueberblick_gremien.tex

```diff
@@ -7,147 +8,414 @@ \section{Überblick im hochschulpolitischen Dschungel}
+den Fachschaften Informatik, Mathematik und Physik haben wir uns dafür
+entschieden, den Fachschaftsrat als lediglich ausführendes Organ zu gestalten,
+die Entscheidungskompetenzen liegen bei der regelmäßig statt findenden
+Fachschaftssitzung.  Das bedeutet, ihr könnt ohne euch extra wählen lassen zu
+müssen jederzeit auf Fachschaftsebene mitarbeiten und mitentscheiden.
+
+Auf zentraler Ebene gibt es in Heidelberg als legislatives Organ den
+Studierendenrat (StuRa).  Er besteht aus VertreterInnen aller
+Studienfachschaften, die von diesen entsandt werden (ca. 64 Personen) sowie aus
+direkt durch die Studierenden gewählten Mitgliedern.  Die Anzahl der direkt
+gewählten Mitglieder skaliert mit der Wahlbeteiligung und liegt zwischen 0
+Personen (bei 0\% Wahlbeteiligung) und ebenfalls 64 Personen (bei 50\% oder
+höherer Wahlbeteiligung).
+
+Der Studierendenrat beschließt als zentrales legislatives Organ über alle
+zentralen Belange der Studierendenschaft (Höhe der Beiträge, Finanzen,
+Wahl der Referate, etc.).
+
+Das exekutive Organ der VS ist in Heidelberg die sogenannte
+„Referatekonferenz“.  Sie besteht aus vom Studierendenrat für bestimmte
+Aufgabenbereiche (z.B.  Kommunales, Ökologie, Hochschulpolitik, Soziales,
+\dots) gewählten ReferentInnen. Diese setzen Beschlüsse des StuRa um und können
+sogar -- sofern der StuRa keine Entscheidung treffen kann oder (z.B. weil zu
+wenige Mitglieder anwesend sind) nicht beschlussfähig ist -- eigenständig im
+Namen der Studierendenschaft entscheiden.
+
+\subsection{Hintergründe zur VS}
+
+\paragraph{Mitgliedschaft in der Verfassten Studierendenschaft}
+
+Alle Studierenden sind qua Immatrikulation automatisch Mitglied der Verfassten
+Studierendenschaft. Weil sie die einzige Möglichkeit zur Partizipation an
+legitimierter Studierendenvertretung ist und ihre Leistungen für alle
+Studierenden offen stehen müssen, ist ein Austritt nicht möglich.
+
+\paragraph{Rechtsfähigkeit}
+
+Bisher ist der „AStA“ (Allgemeiner Studierendenausschuss) nur ein Ausschuss im
+Gesamtgebilde Universität. Durch die Verfasste Studierendenschaft wird die
+Studierendenvertretung zu einer eigenständigen, rechtsfähigen Teilkörperschaft
+innerhalb der Universität. Dadurch kann sie selbst Verträge abschließen und so
+z.B. mit den Verkehrsbetrieben direkt über das Semesterticket verhandeln oder
+Leasing-Fahrzeuge vergünstigt an die Studierenden vermieten.  Diese
+Rechtsfähigkeit gilt allerdings nur für die VS als ganzes, nicht für ihre
+Organe (z.B. Fachschaften).
+
+\paragraph{Beitragshoheit}
+
+Die VS kann von allen Studierenden Beiträge erheben, die zusammen mit dem
+sonstigen Semesterbeitrag (eigentlich genannt Ummeldegebühren) eingezogen
+werden. Mit der Einführung der VS wurde leider nicht festgeschrieben, dass das
+bisherige von der Uni verwaltete Budget des „AStA“, auf das die
+Studierendenvertretung zurückgreift, bestehen bleibt.  Die VS erhebt deshalb
+einen Beitrag von \EUR{7.50} pro Semester. Von diesem werden qua Satzung 40
```

Comment:
Beitrag muss definitiv in zu_aktualisieren. Eigentlich sollte da auch der Sockelbeitrag zum Semesterticket stehen.